### ISSUE TITLE
Unresolved references in Telegram Bot API 7.2 update

### ISSUE THREAD

**Issue by kravetsone** [ark0f/tg-bot-api#20]
The new Telegram Bot API 7.2 objects do not exist in the schema, but other objects reference them

![image](https://github.com/ark0f/tg-bot-api/assets/57632712/888aee45-b1c3-4ecf-a207-776ba6d22eff)
![image](https://github.com/ark0f/tg-bot-api/assets/57632712/20bab067-3ced-422b-a845-db98304cecfb)

**Comment by ENCRYPTEDFOREVER** [comment]
Looks like this is caused by absence of description for these objects...

**Comment by kravetsone** [comment]
> Looks like this is caused by absence of description for these objects...

Yes! We have already discussed this with the author. Would you like to do a PR?)

**Comment by ENCRYPTEDFOREVER** [comment]
> Yes! We have already discussed this with the author.

Any suggestions on fix?

**Comment by kravetsone** [comment]
> > Yes! We have already discussed this with the author.
> 
> Any suggestions on fix?

I'm not familiar with Rust much, so I didn't do it myself. The author suggests leaving an empty string in the absence of a description

Unfortunately, the documentation is poorly done(

I tried to write in @BotTalk chats / bot support, but I got no result...

**Comment by ENCRYPTEDFOREVER** [comment]
I don't think I have time now to do a proper PR but for a dirty fix you can go to **src/extractor.rs**, line 66 and prepend
```rust
else if select_any.peek().matches(&table) {
    State::GetObjectFields {
        name: elem,
        description: RawDescription::default(),
    }
}
 ```
 before the 'else' block.

**Comment by ark0f** [comment]
It looks like issue was fixed after they added descriptions to entities like `BusinessLocation`, `BusinessIntro`, etc 😅

**Comment by nitreojs** [comment]
can confirm, it has been fixed since then. the issue might be closed

**Comment by ENCRYPTEDFOREVER** [comment]
> The issue might be closed

It might reoccur in the future, it's better to implement such case handling...
